pkg/controller/util.go

```diff
@@ -90,6 +90,7 @@ const (
 	vaNodeIDAnnotation         = "csi.alpha.kubernetes.io/node-id"
 )
 
+// SanitizeDriverName sanitize provided driver name.
```

Comment:
sanitize -> sanitizes

Reply:
Done

Reply:
No offense (I really appreciate your work on cleaning up), but this is the kind of comment one gets when the tooling forces one to add something: the comment doesn't add any information that isn't already in the function name and thus adds no value.

I'm now torn between asking for a better comment (for example, one that explains what "sanitize" means in this context) and just ignoring the comment.